sys/mips/signal.c

```diff
@@ -26,15 +27,15 @@ static void stack_unusable(thread_t *td, register_t sp) {
   __unreachable();
 }
 
-int sig_send(signo_t sig, sigaction_t *sa) {
+int sig_send(signo_t sig, sigset_t *return_mask, sigaction_t *sa) {
```

Comment:
Though we diverged from BSD naming for signal functions, I'd still keep signatures as close as possible to their BSD counterparts. So in this case:
``` 
int sig_send(signo_t sig, sigaction_t *sa, sigset_t *mask) ...
```
Please note that we prefer to use short names in function signatures, as BSDs do.

Reply:
... but the signature is different anyway (I'm assuming you're referring to FreeBSD's `sendsig`):
- `sendsig`'s first argument is the handler, not the signal number
- `sendsig`'s second argument is a `ksiginfo` structure, which contains the signal number
So, if we really wanted to "mimic" `sendsig`'s signature, `sigaction_t` should be first and `signo_t` should be second.
Besides, OpenBSD's signature looks like this:
```c
void sendsig(sig_t _catcher, int _sig, sigset_t _mask, const siginfo_t *_si);
```
...so according to that signature, `sig_send`'s signature should look like:
```c
int sig_send(sigaction_t *sa, signo_t sig, sigset_t *mask)
```
or
```c
int sig_send(sigaction_t *sa, sigset_t *mask, signo_t sig)
```
depending on whether you decide to substiture `sig` for the signal number or the `siginfo_t` in the OpenBSD signature.

Reply:
Done.